src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java

```diff
@@ -73,21 +74,35 @@
             capacity.setEndDate(pool.getEndDate());
             capacity.setServiceLevel(getSla(pool));
             Long socketCapacity = getCapacityUnit("sockets", pool);
-            if (products.contains(product) && socketCapacity != null) {
+            if (products.contains(product) && socketCapacity != null && socketCapacity > 0) {
```

Comment:
We do these `x != && x>0` checks a lot in this method both for socketCapacity and coresCapacity.  Would it make sense to pull this out into a quick

```
public boolean isPositive(Long value) {
    return value != null && value > 0;
}
```

It wouldn't shorten things up massively, but it would make the predicates easier to read: `if (products.contains(product) && isPositive(socketCapacity))`.

I feel like this method is probably provided by Commons Lang or Guava or one of those type libraries, but a quick look didn't show anything.

Reply:
Done.